public/stac/HLS-L30.stac.json

```diff
@@ -0,0 +1,624 @@
+          "nodata": 40000,
+          "unit": "degree",
+          "scale": 0.01,
+          "spatial_resolution": 30
+        }
+      ],
+      "title": "Sun Azimuth Angle (SAA)",
+      "roles": [
+        "data"
+      ],
+      "type": "image/tiff; application=geotiff; profile=cloud-optimized"
+    },
+    "SZA": {
+      "raster:bands": [
+        {
+          "data_type": "uint16",
+          "nodata": 40000,
+          "unit": "degree",
+          "scale": 0.01,
+          "spatial_resolution": 30
+        }
+      ],
+      "title": "Sun Zenith Angle (SZA)",
+      "roles": [
+        "data"
+      ],
+      "type": "image/tiff; application=geotiff; profile=cloud-optimized"
+    },
+    "VAA": {
+      "raster:bands": [
+        {
+          "data_type": "uint16",
+          "nodata": 40000,
+          "unit": "degree",
+          "scale": 0.01,
+          "spatial_resolution": 30
+        }
+      ],
+      "title": "View Azimuth Angle (VAA)",
+      "roles": [
+        "data"
+      ],
+      "type": "image/tiff; application=geotiff; profile=cloud-optimized"
+    },
+    "VZA": {
+      "raster:bands": [
+        {
+          "data_type": "uint16",
+          "nodata": 40000,
+          "unit": "degree",
+          "scale": 0.01,
+          "spatial_resolution": 30
+        }
+      ],
+      "title": "View Zenith Angle (VZA)",
+      "roles": [
+        "data"
+      ],
+      "type": "image/tiff; application=geotiff; profile=cloud-optimized"
+    }
+  },
+  "sci:publications": [
+    {
+      "doi": "10.1016/j.rse.2018.09.002",
+      "citation": "Claverie, M., Ju, J., Masek, J. G., Dungan, J. L., Vermote, E. F., Roger, J.-C., Skakun, S. V., & Justice, C. (2018). The Harmonized Landsat and Sentinel-2 surface reflectance data set. Remote Sensing of Environment, 219, 145-161."
+    }
+  ],
+  "title": "Harmonized Landsat Sentinel-2 L30",
+  "extent": {
+    "spatial": {
+      "bbox":[[-180,-90,180,90]]
+    },
+    "temporal": {
+      "interval":[["2013-04-11T00:00:00.000Z",null]]
+    }
+  },
+  "license": "proprietary",
+  "keywords": [
+    "NASA",
+    "USGS",
+    "Landsat",
+    "ESA",
+    "Copernicus",
+    "Sentinel",
+    "Satellite",
+    "Imagery",
+    "Global",
+    "Reflectance",
+    "Temperature"
+  ],
+  "providers": [
+    {
+      "name": "NASA LP DAAC at the USGS EROS Center",
+      "roles": [
+        "producer",
+        "licensor",
+        "processor"
+      ],
+      "url": "https://lpdaac.usgs.gov/"
+    }
+  ],
+
+  "summaries": {
+    "instruments": [
+      "oli",
+      "tiirs",
+      "msi"
+    ],
+    "platform": [
+      "landsat-8",
+      "landsat-9"
+    ],
+    "sci:doi": [
+      "10.5067/HLS/HLSL30.002"
+    ]
+  },
+  "renders": {
+    "truecolor": {
+      "title": "True Color",
+      "assets": [
+        "B04",
+        "B03",
+        "B02"
+      ],
+      "bands_regex": "B[0-9][0-9]",
+      "color_formula": "Gamma+RGB+3.5+Saturation+1.7+Sigmoidal+RGB+15+0.35",
```

Comment:
I don't think we need the **+** but we should make sure the client encode this string correctly

Reply:
https://github.com/vincentsarago/color-operations